$NetBSD: patch-ab,v 1.3 2021/11/27 05:58:50 pho Exp $

- use pkgsrc-provided PREFIX and LOCALEDIR
- fix pthread linkage
- Honor LDFLAGS so that flags like -Wl,-zrelro can be used.

--- Makefile.orig	2005-02-27 13:54:47.000000000 +0000
+++ Makefile
@@ -3,16 +3,16 @@ VERSION = 2.0.7
 CFLAGS = -O2 -std=gnu99 -Wall -fPIC `pkg-config gtk+-2.0 --cflags`
 LIBS = `pkg-config gtk+-2.0 --libs`
 LFLAGS = -shared
-PREFIX = /usr/local
+#PREFIX = /usr/local
 
-LOCALEDIR := $(PREFIX)/share/locale
+#LOCALEDIR := $(PREFIX)/share/locale
 
 ifeq ($(enable_nls),1)
     CFLAGS += -DENABLE_NLS -DLOCALEDIR=\"$(LOCALEDIR)\"
     export enable_nls
 endif
 CFLAGS += -DPACKAGE="\"$(PKGNAME)\""
-export PKGNAME LOCALEDIR
+export PKGNAME #LOCALEDIR
 
 CC = gcc
 
@@ -20,7 +20,7 @@ OBJS = gkrellweather.o
 
 gkrellweather.so: $(OBJS)
 	(cd po && ${MAKE} all )
-	$(CC) $(OBJS) -DVERSION=\"$(VERSION)\" -o gkrellweather.so $(LFLAGS) $(LIBS) -lpthread
+	$(CC) $(OBJS) -DVERSION=\"$(VERSION)\" -o gkrellweather.so $(LFLAGS) $(LDFLAGS) $(LIBS) ${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}
 
 clean:
 	(cd po && ${MAKE} clean )
